### PR TITLE
Keelin/vectornator changes

### PR DESCRIPTION
**Pull request by lucaangeletti**
These are the changes required to get Unsplash working correctly with Vectornator.

### CODE REVIEW

File: UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPicker.swift [hunk 1]
```diff
@@ -83,11 +83,11 @@ extension UnsplashPhotoPicker: UnsplashPhotoPickerViewControllerDelegate {
     func unsplashPhotoPickerViewController(_ viewController: UnsplashPhotoPickerViewController, didSelectPhotos photos: [UnsplashPhoto]) {
         trackDownloads(for: photos)
         photoPickerDelegate?.unsplashPhotoPicker(self, didSelectPhotos: photos)
-        dismiss(animated: true, completion: nil)
+        //dismiss(animated: true, completion: nil)
```
Comment (lucaangeletti) [inline UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPicker.swift:86]: @GweebarraRover Usually we delete the unused code.

File: UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPicker.swift [hunk 2]
```diff
@@ -83,11 +83,11 @@ extension UnsplashPhotoPicker: UnsplashPhotoPickerViewControllerDelegate {
...
     }
 
     func unsplashPhotoPickerViewControllerDidCancel(_ viewController: UnsplashPhotoPickerViewController) {
         photoPickerDelegate?.unsplashPhotoPickerDidCancel(self)
-        dismiss(animated: true, completion: nil)
+        //dismiss(animated: true, completion: nil)
```
Comment (lucaangeletti) [inline UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPicker.swift:91]: @GweebarraRover Same here 👆and also below 👇.

**Review by lucaangeletti — COMMENTED** [review]
Please see my comments.

File: UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPickerViewController.swift [hunk 3]
```diff
@@ -159,22 +158,13 @@ class UnsplashPhotoPickerViewController: UIViewController {
...
-    private func setupNavigationBar() {
-        updateTitle()
-        navigationItem.leftBarButtonItem = cancelBarButtonItem
-
-        if Configuration.shared.allowsMultipleSelection {
-            doneBarButtonItem.isEnabled = false
-            navigationItem.rightBarButtonItem = doneBarButtonItem
-        }
-    }
-
```
Comment (pmattos) [inline UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPickerViewController.swift]: Please let's try to reduce the number of changes... can we leave this method as is?

File: UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPickerViewController.swift [hunk 4]
```diff
@@ -199,10 +189,6 @@ class UnsplashPhotoPickerViewController: UIViewController {
         ])
     }
 
-    private func setupPeekAndPop() {
-        previewingContext = registerForPreviewing(with: self, sourceView: collectionView)
-    }
-
```
Comment (pmattos) [inline UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPickerViewController.swift:205]: Please let's try to reduce the number of changes... can we leave this method as is?

**Review by pmattos — CHANGES_REQUESTED** [review]

**Comment by pmattos** [comment]
@GweebarraRover Please check my review comments...

**Comment by pmattos** [comment]
@GweebarraRover @lucaangeletti To be clear guys, we need to merge this *before* [PR 337](https://github.com/LinearityGmbH/Vectornator/pull/337)... otherwise the submodule commit (ie, for `Frameworks/UnsplashPhotopicker`) won't reflect the right one. The submodule needs to point to the commit created by merging this PR into the `master` branch...

**Comment by GweebarraRover** [comment]
Yes, I will revert the deleted code and get this PR updated. Then it's simply pointing the unsplash branch submodule to this and it will be finished.

**Pull request merged**